packages/react/src/hooks/useSignUp.ts

```diff
@@ -24,6 +26,10 @@ export const useSignUp: UseSignUp = () => {
   const isomorphicClerk = useIsomorphicClerkContext();
   const client = useClientContext();
 
+  useEffect(() => {
+    isomorphicClerk.telemetry?.record(eventMethodCalled('useSignUp'));
+  }, [isomorphicClerk.telemetry]);
```

Comment:
Based on the new cache mechanism, i believe we can skip the usage of useEffect

Reply:
Makes sense. Even without the `useEffect`, it'll only record the event once for multiple function executions. 

Removed on https://github.com/clerk/javascript/pull/3341/commits/e41aaad55ee6b22573e6d1ba11d3a7a76085af80